frontend/src/metabase/nav/components/PaymentBanner/PaymentBanner.tsx

```diff
@@ -0,0 +1,54 @@
+import { jt, t } from "ttag";
```

Comment:
This should look mostly like the previous version of the `AppBanner`
https://github.com/metabase/metabase/blob/fdc07959b9369547221ac5ab897a951f24160f46/frontend/src/metabase/components/AppBanner/AppBanner.jsx

The only notable change is the logic to determine whether we should display this banner is lifted up, so we can decide when to display this banner or the database prompt banner.